paymentsheet/src/test/java/com/stripe/android/paymentsheet/LinkHandlerTest.kt

```diff
@@ -88,55 +90,53 @@ class LinkHandlerTest {
 
     @Test
     fun `Sets up state correctly for user that fails verification`() = runLinkTest {
+        accountStatusFlow.emit(AccountStatus.Error)
         handler.setupLinkLaunchingEagerly(
             testScope,
             LinkState(configuration, LinkState.LoginState.NeedsVerification),
         )
+
         assertThat(handler.isLinkEnabled.first()).isTrue()
         assertThat(handler.activeLinkSession.first()).isFalse()
-        assertThat(handler.linkConfiguration.first()).isEqualTo(configuration)
+        assertThat(accountStatusTurbine.awaitItem()).isEqualTo(AccountStatus.Error)
         processingStateTurbine.ensureAllEventsConsumed()
         assertThat(handler.showLinkVerificationDialog.first()).isFalse()
-        verifyNoInteractions(linkLauncher)
 
         handler.handleLinkVerificationResult(false)
 
         assertThat(handler.activeLinkSession.first()).isFalse()
         assertThat(handler.showLinkVerificationDialog.first()).isFalse()
-
-        verifyNoInteractions(linkLauncher)
     }
 
     @Test
     fun `Sets up state correctly for null LinkState`() = runLinkTest {
         handler.setupLinkLaunchingEagerly(testScope, null)
         assertThat(handler.isLinkEnabled.first()).isFalse()
         assertThat(handler.activeLinkSession.first()).isFalse()
-        assertThat(handler.linkConfiguration.first()).isNull()
         assertThat(handler.showLinkVerificationDialog.first()).isFalse()
+        accountStatusTurbine.expectNoEvents()
```

Comment:
Should we instead replace this with `ensureAllEventsConsumed` at the end of `runLinkTest` along side `processingStateTurbine.ensureAllEventsConsumed()`